libs/core/src/lib/date-picker/date-picker.component.html

```diff
@@ -71,6 +71,14 @@
             [previousButtonDisableFunction]="previousButtonDisableFunction"
             [nextButtonDisableFunction]="nextButtonDisableFunction"
         ></fd-calendar>
+
+        <div *ngIf="showTodayButton" fd-bar barDesign="footer">
+            <div fd-bar-right>
+                <fd-bar-element>
+                    <button fd-button label="Today" (click)="onTodayButtonClick()"></button>
```

Comment:
We should (for now) provide an input for the label value that defaults to "Today" so the developer can pass their own strings in different languages etc